tests/test_cases/test_cocotb/test_edge_triggers.py

```diff
@@ -208,3 +209,18 @@ async def wait_ten():
     b = cocotb.fork(wait_ten())
     await a.join()
     await b.join()
+
+
+@cocotb.test(timeout_time=100, timeout_unit='ns',
+             expect_error=SimTimeoutError if cocotb.LANGUAGE in ["verilog"] and cocotb.SIM_NAME.lower().startswith(("riviera", "aldec")) else ())   # gh-2344
```

Comment:
This is what black says if I add in parens.
```suggestion
@cocotb.test(
    timeout_time=100,
    timeout_unit="ns",
    expect_error=(
        SimTimeoutError if (
            cocotb.LANGUAGE in ["verilog"]
            and cocotb.SIM_NAME.lower().startswith(("riviera", "aldec"))
        )
        else ()
    ),
)  # gh-2344
```

Reply:
Of course flake8 doesn't like it =/

Reply:
moved the `and`. It's still clearer than it was and flake8 isn't complaining.